### PR TITLE
added project files

### PR DESCRIPTION
**Pull request by shafiuyushawu**
### Hello Reviewer ✋ 

# **Features implemented: ✍️**

## 1. API

- Chose [Weather API](https://www.weatherapi.com/docs/)

## 2. Design

 - Imitated the design presented by Microverse authored by  Nelson Sakwa.
 - Acknowledged author in Readme.

### 3.1. Homepage

- When the page loads, the web app renders current weather conditions in your locality, list of next predictable 10 days in your locality.
- Users can also search for other geographical weather locations by Country, Region, Cities, etc.,
- When the user clicks or taps on the weather detail card, it routes them to detail page, along with detail weather conditions from the API

### 3.2. Details page

 - Extracted data need to be rendered from the API to redux store
 - When using Click on the home button it re-routes the user back to home.

## 4. Testing components.
- Wrote test to create snapshots of components and test if components are rendered.

## 5. Technical requirement
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API is stored in the Redux store.
- Data is filtered from the API using a Filter stateless component.
- Every page (main and details) has a unique route within the SPA.
- Project deployed using Render. [Live link](https://weather-app-r266.onrender.com/)

## 6. Project presentation
- [Video presentation](https://www.loom.com/share/c55f339be63c48ee9f3d4b83a35c6cfd)

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (BenMukebo) [inline src/index.css:3]: - [x] Hi @shafiuyushawu! The external API is to have data you can display on your pages, however, according to the design requirement:

image

According to the design, these are the possible colors and fonts:

image

You have the font right, however, the colors don't match. Please consider changing it to meet this requirement! 🤓 🔧 

|Template|Your design|
|---|---|
| image | image|
| image |image



Also, If you look closely, in your home page, the elements, in your case teams have a "chess-like" color design, if the left side is light, the right side should be darker and the inverse. Please consider applying this as suggested by the project requirements! 🤓 🔧

**Review by BenMukebo — CHANGES_REQUESTED** [review]
## Hi @shafiuyushawu, 

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


### To Highlight 🎉
- Git flow is followed. :heavy_check_mark:
- You have followed  JavaScript best practices. ✔️ 
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- Good presentation video. ✔️ 
- Your app has been well deployed using to Heroku, Netlify or GitHub Pages. ✔️ 

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (shafiuyushawu) [reply src/index.css:3]: Resolved, 
Thanks for the comments

**Review by TadesseDev — APPROVED** [review]
### Hi @shafiuyushawu 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shafiuyushawu** [comment]
@TadesseDev 
Thank you

**Pull request merged**